src/python/pants/backend/docker/goals/tailor.py

```diff
@@ -29,19 +31,22 @@ class PutativeDockerTargetsRequest(PutativeTargetsRequest):
 async def find_putative_targets(
     req: PutativeDockerTargetsRequest, all_owned_sources: AllOwnedSources
 ) -> PutativeTargets:
-    all_dockerfiles = await Get(Paths, PathGlobs, req.search_paths.path_globs("Dockerfile"))
+    all_dockerfiles = await Get(Paths, PathGlobs, req.search_paths.path_globs("Dockerfile*"))
```

Comment:
Looks like this should be `*Dockerfile*`. But imo, save this for a dedicated followup? This PR is already pretty big. It'd be better to avoid any diff in `tailor.py` if possible

Reply:
Agree. Punting the tailor changes, as we're backwards compatible, they're not required.